platipy/imaging/label/comparison.py

```diff
@@ -293,3 +293,94 @@ def compute_metric_hd(label_a, label_b, auto_crop=True):
     hausdorff_distance_value = hausdorff_distance.GetHausdorffDistance()
 
     return hausdorff_distance_value
+
+
+def compute_metric_total_apl(label_ref, label_test, distance_threshold_mm=0):
```

Comment:
Do you think 0 is a good default value here? For the surface dsc I chose 3mm (that seems to be a common value). In practice you mentioned IoV MDA would be the correct value to choose... but I wonder if 3mm is a reasonable estimate for "most" cases?

Reply:
Good thought. I'm a bit conflicted, as the APL was originally designed to compare autocontours to edited autocontours - i.e. most of the contour would be identical (so distance = 0) and in such cases one would want to measure any change made. If you chose 3mm for the surface dice then I think being consistent and using 3mm here would be good!